docs/configuration.asciidoc

```diff
@@ -258,12 +258,12 @@ server versions.
 
 [float]
 [[config-ignore-urls]]
-=== `ELASTIC_APM_IGNORE_URLS`
+=== `ELASTIC_APM_TRANSACTION_IGNORE_URLS`
 
 [options="header"]
 |============
-| Environment               | Default | Example
-| `ELASTIC_APM_IGNORE_URLS` |         | `/heartbeat*, *.jpg`
+| Environment                           | Default | Example
+| `ELASTIC_APM_TRANSACTION_IGNORE_URLS` |         | `/heartbeat*, *.jpg`
 |============
 
 A list of patterns to match HTTP requests to ignore. An incoming HTTP request
```

Comment:
Can you please add a note below that we'll continue to support "ELASTIC_APM_IGNORE_URLS", and that it's deprecated and will be removed in a future release